### PR TITLE
Fix broken tests, add rubygems support in travis

### PR DESCRIPTION
**Pull request by usbsnowcrash**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,14 +2,20 @@ language: ruby
...
-- 2.3.1
-- 2.2
-- 2.1
+- 2.3
 notifications:
   email: false
   hipchat:
     rooms:
-      secure: g6Ve1qDiVR9VHOJwXVr4iXEMZNfLUL2iBlGzF9XVNySJxYmPOpfOhb+GjN2vefTvloZVgRvTeXE/9rldk6OPseXqMCuLP8wbXm3TeKQDhsjeyWQt4oXUKk5PePZK83PJiv7VnW09eRIwPkUhFWOeCV8qZxREHseyxz8VZllfUN4=
+      secure: KbZjr/brG9QE4jeqOXHXlDDmnLRy/yE8flTtv9xsKn5jUxJXMXp8NO37Yl/K2L957J2xbI/U1p6kABM5oi0W5MGtlu5yLrzNoA97YecUs7z/ynRVHf3FcFM5VhcmVTQI89cZ2Z+/oj0DBuodc+dEo4Qs61LTwEKZCOijAxzQDQo=
```
Comment (bowersaa) [inline .travis.yml]: Where did it alert before?

Comment (usbsnowcrash) [inline .travis.yml]: consumer platform room, Ive got it going to consumer-main with this change

Comment (bowersaa) [inline .travis.yml]: The more appropriate place

**Comment by tkling** [comment]
screen shot 2017-07-10 at 4 09 39 pm
Woot!

**Pull request merged**